libs/wingc/src/type_check.rs

```diff
@@ -2458,24 +2460,29 @@ impl<'a> TypeChecker<'a> {
 			.take_while(|arg| arg.typeref.is_option())
 			.count();
 
+		// ^ in min_parameters we count more optionals, why the difference?
+
 		// Verify arity
+		// ^ 2 header comments with same name in the same function, maybe clarify their difference
```

Comment:
Good points.
1. I'm not sure and there might be a bug here. @MarkMcCulloh I think you made the changes in `min_parameters`, perhaps you can take a look? @Lancear maybe you can try to recreate a case where they differ using a struct for named args and see what happens?
2. Right, feel free to clarify the comment.

Reply:
I think I managed to fix it in:
https://github.com/winglang/wing/pull/4300